tests/ptable/test_ptable_matplotlib.py

```diff
@@ -22,7 +22,7 @@
 
 @pytest.fixture
 def glass_elem_counts(glass_formulas: pd.Series[Composition]) -> pd.Series[int]:
-    return pmv.count_elements(glass_formulas)
+    return pmv.count_elements(glass_formulas, fill_value=0)
```

Comment:
This is added to better match the type annotation (`pd.Series[int]`), for a "counting" task, using zero for missing element may be more appropriate.